spring-context/src/test/resources/org/springframework/beans/factory/xml/XmlBeanFactoryTests-reftypes.xml

```diff
@@ -14,6 +14,28 @@
 		<property name="spouse" ref="jenny"/>
 	</bean>
 
+	<bean id="jennyc"  class="org.springframework.tests.sample.beans.TestBean">
+		<constructor-arg index="1"><ref bean="davidc"/></constructor-arg>
+	</bean>
+
```

Comment:
I'm guessing `c` stands for constructor and `d` stands for depends-on, but what does `s` stand for?

In any case, I'd recommend more meaningful names -- for example, perhaps with suffixes such as `_constructor`, `_depends_on`, and `_prototype`.